docs/website/docs/community/blog/posts/mmt4d.md

```diff
@@ -63,7 +63,7 @@ future, and in this case, it let us spring board off of high quality code
 generation which was already well integrated and composed well with other
 features of the compiler.
 
-## Existing Matrix Multplication Code Generation
+## Existing Matrix Multiplication Code Generation
```

Comment:
:O

found another case of this typo (with a bonus typo of `matrix` too!): https://github.com/iree-org/iree/blob/9a294eb6829430b5005cd305e63046dbbf06660f/compiler/src/iree/compiler/Preprocessing/Common/ConvertConv2DToImg2Col.cpp#L78-L79